.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Plain JF Dotnet 8",
+    "image":"mcr.microsoft.com/devcontainers/dotnet:8.0-jammy",
+        //required to get skia to run
+	"postCreateCommand": "sudo apt-get update && sudo apt-get install -y libfontconfig1",
```

Comment:
Use features instead so all apt data is cleared afterwards. Like this:

"ghcr.io/devcontainers-contrib/features/apt-packages:1": {
      "preserve_apt_list": false,
      "packages": ["libfontconfig1"]
    }